extension/flat_tensor/serialize/serialize.cpp

```diff
@@ -149,10 +173,10 @@ runtime::Error save_ptd(
       padding_required(
           FlatTensorHeader::kHeaderExpectedLength, tensor_alignment));
 
-  // Write flatbuffer
+  // Write flatbuffer, offset by 8 bytes since we wrote those before the header.
```

Comment:
Some specifics to make it more clear
```suggestion
  // Write flatbuffer, offset by 8 bytes (4-byte root table offset + 4-byte file
  // identifier) since we wrote those before the FlatTensorHeader.
```